components/auth/auth-modal.tsx

```
@@ -0,0 +1,68 @@
+          </Button>
+        </div>
+        <DialogFooter className="flex justify-center">
+          <p className="text-sm text-gray-600">
+            New here?{" "}
+            <button 
+              className="text-blue-600 hover:underline"
+              onClick={() => {
+                // TODO: Implement account creation
+                console.log("Create your space clicked")
+              }}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)